test/integration/test_datatype_upload_irods.py

```diff
@@ -0,0 +1,107 @@
+
+import os
+import string
+import subprocess
+import tempfile
+import time
+
+import pytest
+
+from galaxy_test.driver import integration_util
+from .test_datatype_upload import (TEST_CASES, upload_datatype_helper, UploadTestDatatypeDataTestCase)
```

Comment:
```suggestion
from .test_datatype_upload import (
    temp_file
    TEST_CASES,
    upload_datatype_helper, 
    UploadTestDatatypeDataTestCase
)
```

but please also check if you can move the temp_file fixture to conftest.py file in `test/integration`. If you do that they will be injected as need and no import is necessary.